packages/components/src/styles/components/side-nav/main.scss

```
@@ -8,25 +8,13 @@
 //
 
 .hds-side-nav {
```

I would leave a `position: relative` here, so the `SideNav::ToggleButton` position is determined in relation to this component, not its parent.
```suggestion
.hds-side-nav {
  position: relative;
```